features/full_tests/async_error_flush.feature

```diff
@@ -1,20 +0,0 @@
-Feature: Flushing errors does not send duplicates
```

Comment:
It seems preferable to use a `@FlakyTest` tag as that would result in less merge conflicts and would be easy to add if new flakes are discovered

Reply:
Yes, that's a good idea and occurred to me after creating the PR.  I needed to raise tickets for and comment the individual tests anyway, so can factor this into that - in a fast-following PR.

Reply:
Actually, I think I'll do it on this PR.

Reply:
Now done.